.github/workflows/deploy.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.9, "3.10", "3.11", "3.12"]
+        python-version: ["3.11", "3.12"]
```

Comment:
`python 3.13` is not being deployed. Is that on purpose?

Reply:
I believe it should, and I forgot to mention.

Thanks @renatomello for noticing it!